tox.ini

```
@@ -42,38 +42,35 @@ deps =
     devdeps: pandas>=0.0.dev0
     devdeps: pluggy>=0.0.dev0
     devdeps: pytest>=0.0.dev0
-    # Oldest deps we pin against
-    oldestdeps: asdf-astropy<0.1.2
-    oldestdeps: asdf<2.9.0
-    oldestdeps: astropy<5.1.0
-    oldestdeps: beautifulsoup4<4.9.0
-    oldestdeps: cdflib<0.3.20
-    oldestdeps: dask[array]<2021.5.0
-    oldestdeps: drms<0.7
+    # oldestdeps we pin against to ensure correct versions
+    oldestdeps: asdf-astropy<0.3.0
+    oldestdeps: asdf<2.12.0
+    oldestdeps: astropy<5.3.0
+    oldestdeps: beautifulsoup4<4.12.0
+    oldestdeps: cdflib<0.4.5
+    oldestdeps: dask[array]<2022.6.0
+    oldestdeps: drms<0.8
     oldestdeps: glymur<0.9.2
-    oldestdeps: h5netcdf<0.12
+    oldestdeps: h5netcdf<1.1.0
     oldestdeps: matplotlib<3.6.0
     oldestdeps: numpy<1.22.0
-    oldestdeps: opencv-python<4.8.0.72
-    oldestdeps: pandas<1.3.0; platform_system!='Darwin' or platform_machine!='arm64'
-    # pandas 1.4.0 is the first with wheels for macOS arm64
-    oldestdeps: pandas<1.5.0; platform_system=='Darwin' and platform_machine=='arm64'
+    oldestdeps: opencv-python<4.7.0.68
+    oldestdeps: pandas<1.5.0
     oldestdeps: parfive<1.3.0
```

```suggestion
    oldestdeps: parfive<2.1.0
```
Since the min pin is 2.0.0 in `setup.cfg`